pkg/sbom/scanner/scanner_test.go

```diff
@@ -199,6 +199,8 @@ func TestRetryLogic_ImageDeleted(t *testing.T) {
 		}
 	}, 15*time.Second, 1*time.Second)
 	cancel()
+	// Ensure the collector is stopped
+	shutdown.WaitUntil(time.After(2 * time.Second))
```

Comment:
Can we get the collectors status?
It would be nice if collectors have a IsStoped() function